themes/default/layouts/registry/api.html

```diff
@@ -19,7 +19,7 @@ <h1>{{ .Params.title }}: API Docs</h1>
                     {{ .Content }}
                 </section>
             </div>
-            <div class="w-3/12 pr-8 mb-2 pt-8 lg:order-first lg:pt-0 lg:mt-0">
+            <div class="flex-shrink-0 pr-8 mb-2 pt-8 lg:order-first lg:pt-0 lg:mt-0">
```

Comment:
I moved this to using `flex-shrink` since we moved to using flex to layout the page with the new registry changes that were shipped. When we were using the 12 column grid to determine the width of this column, the width was actually being determined in two places - both here (e.g. `w-3/12`) and in the stencil component based on the character limitations set at different breakpoints. This ended up resulting in some awkward extra white space at certain screen widths when I updated the stencil component to remove the breakpoint between 1280 and 1536. Now the column will just shrink to the width needed based on the character count set in the stencil component and the main content area will naturally expand to fill the remaining space on the screen.